### ISSUE TITLE
java.io.FileNotFoundException: Cidaas.xml

### ISSUE THREAD

**Issue by DDihanov** [cidaas/cidaas-android-sdk#16]
Hey I am getting an error ```Property file is missing.``` when I try to make a call. I tracked the issue and it boils down to  ```java.io.FileNotFoundException: Cidaas.xml``` This occurs with the new 2.0.9 and 2.1.0 SNAPSHOT versions. The old 2.0.2 version does not have this issue when the file is named "Cidaas.xml".

I have tracked down the error to here:

https://github.com/Cidaas/cidaas-sdk-android/blob/e0695728b94a74bdfa562134701cc429931f0782/cidaas/src/main/java/de/cidaas/sdk/android/helper/general/FileHelper.java#L56

[In the documentation it says the file name should be "cidaas.xml"](https://github.com/Cidaas/cidaas-sdk-android#note--the-file-name-must-be-cidaasxml), but the FileHelper tries to read a "Cidaas.xml". I have tried with both "cidaas.xml" AND "Cidaas.xml" but it still crashes.

**Comment by cidaas** [comment]
Hello,

can you try this with the latest master-SNAPSHOT version?

```
dependencies {
	        implementation 'com.github.Cidaas:cidaas-sdk-android:master-SNAPSHOT'
	}
```

**Comment by DDihanov** [comment]
> Hello,
> 
> can you try this with the latest master-SNAPSHOT version?
> 
> ```
> dependencies {
> 	        implementation 'com.github.Cidaas:cidaas-sdk-android:master-SNAPSHOT'
> 	}
> ```

Hey it seems to be working with ```master-SNAPSHOT```. I tried both ```com.github.Cidaas:cidaas-v2-sdk-android:master-SNAPSHOT``` and ```com.github.Cidaas:cidaas-sdk-android:master-SNAPSHOT``` and they both work with ```cidaas.xml```.

I should be using the ```v2``` one right? 🤔

**Comment by cidaas** [comment]
No, please use the one without the -v2.  
In the current refactoring process, we have removed the version number from the name.

**Comment by cidaas** [comment]
We have released the version 3.0.0: 'com.github.Cidaas:cidaas-android-sdk:3.0.0' including this fix.
https://jitpack.io/#Cidaas/cidaas-android-sdk/3.0.0  

I am going to close this issue as the problem is fixed now. If it reoccurs, or any other problems occur, feel free to reopen this or create a new issue.
